eth/core/state/statedb.go

```diff
@@ -230,10 +230,6 @@ func (sdb *stateDB) GetStorageProof(_ common.Address, _ common.Hash) ([][]byte,
 	return nil, nil
 }
```

Comment:
`GetStorageProof` also needs to be deleted so it calls the Plugin

Reply:
reeeeee its so cooked LMFAO we need MPT